apps/editing-toolkit/editing-toolkit-plugin/coming-soon/test/class-coming-soon-test.php

```diff
@@ -121,13 +121,13 @@ public function test_add_option_to_new_site_with_coming_soon_meta() {
 		$result = add_option_to_new_site( get_current_blog_id(), null, null, null, null, $meta );
 
 		$this->assertTrue( $result );
-		$this->assertEquals( 1, get_option( 'wpcom_public_coming_soon' ) );
+		$this->assertSame( 1, get_option( 'wpcom_public_coming_soon' ) );
 
 		// Check that we've added the action correctly.
 		self::delete_coming_soon_site_options();
-		$this->assertEquals( 0, get_option( 'wpcom_public_coming_soon' ) );
+		$this->assertFalse( get_option( 'wpcom_public_coming_soon' ) );
```

Comment:
@ramonjd @andrewserong, howdy! PHPCS had me change `assertEquals` to `assertSame`, and it turns out that `get_option( 'wpcom_public_coming_soon' )` was returning `false` rather than `0`. (Thanks PHPCS!) I think this might be because no option exists, in which case WP returns false.

I've changed this assertion (and the one on line 155) to `assertFalse` instead. I wanted to check that this change is practically equivalent and that no changes to the actual code are required.

Reply:
Thanks @noahtallen, yes this should be `assertFalse`, the actual code doesn't need to be changed. Thanks for fixing it up!

Reply:
> think this might be because no option exists

You're right. Thanks for picking this up.

Just checked in the shell on a site after having deleted the option:

```
var_dump( get_option( 'wpcom_public_coming_soon' );
```
Thank you!!!

Reply:
🙇